util/fpga/cw_spiflash.py

```diff
@@ -135,25 +135,35 @@ class _Bootstrap:
                                            'PIN_CS',
                                            'PIN_TRST',
                                            'PIN_SRST',
-                                           'PIN_JTAG_SPI',
-                                           'PIN_BOOTSTRAP'])
+                                           'PIN_SW_STRAP0',
+                                           'PIN_SW_STRAP1',
+                                           'PIN_SW_STRAP2',
+                                           'PIN_TAP_STRAP0',
+                                           'PIN_TAP_STRAP1'
+                                           ])
     _PIN_MAPPINGS = {}
-    _PIN_MAPPINGS['CW305'] = PinMapping('USB_A13',
+    _PIN_MAPPINGS['CW305'] = PinMapping('USB_A9',
```

Comment:
i'm a bit confused, where are all these `USB_*` naming defined?

Reply:
hmmm i guess it's from the cw library itself... i need to find it and have a look.

Reply:
I do believe so, yes. I see these connections on the reference schematic and XDC files.

Reply:
I think opentitantool needs to be updated as well, I haven't looked in detail where these pins are defined yet but it looks like `sw/host/opentitanlib/src/transport/cw310/mod.rs` and `do_update` in `sw/host/opentitanlib/src/bootstrap/mod.rs` need to be updated.

Reply:
Thanks @alphan. I'll take a look at the OTT parts also.

Indeed, on the device side we can just continue to sample one pin and implement the more elaborate decoding later.